docs/gadget-devel/gadget-intro.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Gadget Introduction
+sidebar_position: 100
+description: Introduction to Gadgets
+---
+
+Gadgets are the central component in the Inspektor Gadget framework. A Gadget is
+an [OCI image](https://opencontainers.org/) that includes one or more eBPF
+programs, metadata YAML file and, optionally, WASM modules for post processing,
+etc. As OCI images, they use the same tooling as containers and share the same
```

Comment:
"etc."? What else can be there? I would remove it.

Reply:
Ideas so far:
- Logo
- Documentation (Readme file)
- Source code
- Tests
 
I know we aren't using them now, but I'm sure we'll have more things there soon.